controllers/userController.js

```diff
@@ -125,18 +127,7 @@ module.exports = {
         console.log(`Error updating user by ID: ${error.message}`);
         next(error);
       });
-  },
-  delete: (req, res, next) => {
-    let userId = req.params.id;
-    User.findByIdAndRemove(userId)
-      .then(() => {
-        res.locals.redirect = "/user/login";
-        next();
-      })
```

Comment:
we may need it for admin user (just comment it out instead pleeeease)

Reply:
all the stuff that looks like it was deleted i actually just commented out and moved down in the code :D that's why it looks like it was removed i guess...